r/vignettes/developers/setup.Rmd

```diff
@@ -281,11 +281,10 @@ withr::with_makevars(list(CPPFLAGS = "", LDFLAGS = ""), remotes::install_github(
   environment variables that determine how the build works and what features
   get built.
 * `TEST_OFFLINE_BUILD`: When set to `true`, the build script will not download
-  prebuilt the C++ library binary.
+  prebuilt the C++ library binary or, if needed, `cmake`.
```

Comment:
```suggestion
  prebuilt the C++ library binary or, if needed, `cmake` of an appropriate version.
```

We could also say "`cmake` >= 3.16" here instead, but then have to make sure we bump both. But it might be nice to call out the version requirement here too.

Reply:
I don't feel strongly; don't think it matters much one way or another. Most likely if you're reading this file, you've already tried to install and hit the error message that tells you what version you need 🤷 

IMO your exact suggestion isn't worth adding, I'd think that appropriate version was implied. If you wanted to list the version here, I'd leave a comment in nixlibs.R next to `cmake_minimum_version` saying to update the vignette when you bump it. (Of course, that only gets bumped when we bump https://github.com/apache/arrow/blob/main/cpp/CMakeLists.txt#L18, and really only if anyone remembers that we have it listed here too.)